### PR TITLE
MAM-4132-improvements-to-webview

### PR DESCRIPTION
**Pull request by derspyy**

### CODE REVIEW

**Comment by derspyy** [comment]
making this a draft because i want to look into a crash.

**Review by bnolens — CHANGES_REQUESTED** [review]
Please check my notes

File: Mammoth/Views/Cells/PostCardCell/PostCardWebview.swift [hunk 1]
```diff
@@ -194,9 +200,12 @@ extension PostCardWebview {
             let inverse_ratio = Double(iframe.height) / Double(iframe.width)
             stackHeightConstraint = mainStackView.heightAnchor.constraint(equalTo: mainStackView.widthAnchor, multiplier: inverse_ratio)
             stackHeightConstraint?.isActive = true
-            
+            var placeholder: UIImage? = nil
+            if let blurhash = linkCard.blurhash {
+                placeholder = UnifiedImage(blurHash: blurhash, size: .init(width: 32, height: 32))
```
[the remark this answers is not in the document]
Comment (derspyy) [reply Mammoth/Views/Cells/PostCardCell/PostCardWebview.swift:205]: link card blurhashes aren't taken into account in that function, though i'll add it. thanks!

**Review by bnolens — APPROVED** [review]
LGTM

**Pull request merged**